Source/WebCore/rendering/RenderBlockFlow.cpp

```diff
@@ -593,6 +593,9 @@ void RenderBlockFlow::layoutBlock(bool relayoutChildren, LayoutUnit pageLogicalH
             relayoutChildren = true;
         layoutPositionedObjects(relayoutChildren || isDocumentElementRenderer());
     }
+
+    updateDescendantTransformsAfterLayout();
```

Comment:
Are there classes other than RenderBlockFlow that need this?

Reply:
Yeah I think the other subclasses of RenderBlock need this. Will add some tests on grid etc.

Reply:
Added to RenderBlock subclasses and added tests for grid and flex.